RecCalorimeter/include/RecCalorimeter/ClusterJet.h

```diff
@@ -0,0 +1,65 @@
+#ifndef RECCALORIMETER_CLUSTERJET_H
+#define RECCALORIMETER_CLUSTERJET_H
+
+// std
+#include <map>
+
+// Gaudi
+#include "GaudiAlg/GaudiAlgorithm.h"
+
+#include "fastjet/JetDefinition.hh"
+
+// EDM4hep
+#include "edm4hep/ClusterCollection.h"
+
+
+namespace k4::recCalo{
+
+class ClusterInfo : public fastjet::PseudoJet::UserInfoBase{
+ public:
+ ClusterInfo(const int & index) : _index(index){}
+  int index() const{ return _index; }
+ protected:
+  int _index;
+};
+
+
+
+/** @class ClusterJet k4RecCalorimeter/RecCalorimeter/src/components/ClusterJet.h
+ *
+ *  @author Jennifer Roloff
+ */
+
+class ClusterJet {
+public:
+  ClusterJet(std::string jetAlg, double jetRadius, int clustering = 0, double minPt = 0, int clusterArgs = 0);
```

Comment:
Either here or in the main description above, can you specify that `int clustering = 0` refers to exclusive/inclusive clustering? This is not easy to guess

Reply:
I've changed the variable name to match the other tools, which also makes its definition clearer.